src/EditCell.tsx

```diff
@@ -1,37 +1,38 @@
 import { useState, useCallback } from 'react';
+import { createPortal } from 'react-dom';
 import { css } from '@linaria/core';
 
-import EditorContainer from './editors/EditorContainer';
+import { useMouseDownOutside } from './hooks';
 import { getCellStyle, getCellClassname } from './utils';
-import type { CellRendererProps, SharedEditorProps, Omit } from './types';
+import type { CellRendererProps, SharedEditorProps } from './types';
 
 const cellEditing = css`
   padding: 0;
 `;
 
-const cellEditingClassname = `rdg-cell-editing ${cellEditing}`;
+// TODO: remove 1 of the two classes
+const cellEditingClassname = `rdg-cell-editing rdg-editor-container ${cellEditing}`;
 
 type SharedCellRendererProps<R, SR> = Pick<
   CellRendererProps<R, SR>,
   'rowIdx' | 'row' | 'column' | 'colSpan'
 >;
 
-interface EditCellProps<R, SR>
-  extends SharedCellRendererProps<R, SR>,
-    Omit<React.HTMLAttributes<HTMLDivElement>, 'style' | 'children'> {
+interface EditCellProps<R, SR> extends SharedCellRendererProps<R, SR> {
   editorProps: SharedEditorProps<R>;
+  onKeyDown: Required<React.HTMLAttributes<HTMLDivElement>>['onKeyDown'];
```

Comment:
`EditCell` is a strictly internal component with only specific props.

Reply:
Should we also flatten the props and removed `editorProps`?